utils/parsers/shows.js

```diff
@@ -6,19 +6,20 @@ dayjs.locale('es')
 
 const parseShows = movieList => {
   const showsParsed = movieList
-    .map(
-      ({ format, version, cinemaList }) =>
-        cinemaList.map(({ id: cinemaId, sessionList }) =>
-          sessionList.map(({ id: sessionId, feature: featureId, dtm: timestamp }) => ({
-            timestamp,
-            date: titleize(dayjs(timestamp).format('dddd D [de] MMMM')),
-            time: dayjs(timestamp).format('HH[:]mm'),
-            link: createTicketsLink(cinemaId, sessionId, featureId),
-            format: format.toUpperCase(),
-            version: titleize(version),
-          }))
-        )[0]
-    )
+    .map(({ format, version, cinemaList }) =>
+      cinemaList.map(({ id: cinemaId, sessionList }) => {
+        const shows = sessionList.map(({ id: sessionId, feature: featureId, dtm: timestamp }) => ({
+          timestamp,
+          date: titleize(dayjs(timestamp).format('dddd D [de] MMMM')),
+          time: dayjs(timestamp).format('HH[:]mm'),
+          link: createTicketsLink(cinemaId, sessionId, featureId),
+          format: format.toUpperCase(),
+          version: titleize(version),
+        }))
+
+        return { cinemaId: String(cinemaId), shows }
+      })
+    )[0]
```

Comment:
If you are looking for only the first element, try doing this[0] in the movieList before the `map`, otherwise you'll be iterating all around the list to discard most of the items a bit later

Reply:
Well from your comment I realize I was picking only the first format function i.e:
<img width="275" alt="Captura de Pantalla 2019-06-26 a la(s) 14 14 26" src="https://user-images.githubusercontent.com/2475912/60200574-c437ac80-981c-11e9-9937-36b35a16d2a9.png">
And parsing only the shows of the first format only.

So in order to fix this I left the following structure:
```javascript
[
  {
    "formatId": 3,
    "format": "2d Premium",
    "version": "Subtitulada",
    "cinemaWithShows": {
      "104": [] // all shows data here like seats, date, time, etc
      "110": [] // all shows data here like seats, date, time, etc
    }
  },
  {
    "formatId": 4,
    "format": "2d Premium",
    "version": "Castellano",
    "cinemaWithShows": { } // more shows
  }
]
```
This was my best approach surely we can improve it later but now they are grouped by format and inside each one we have a `cinemaWithShows` object/map with keys being the `cinemaId` and with and array of the shows as value.

`/shows/:movieId`
<img width="1400" alt="Captura de Pantalla 2019-06-26 a la(s) 14 12 12" src="https://user-images.githubusercontent.com/2475912/60200854-5475f180-981d-11e9-9b70-812d670d23fb.png">

`/shows/:movieId/:cinemaId`
<img width="1393" alt="Captura de Pantalla 2019-06-26 a la(s) 14 12 36" src="https://user-images.githubusercontent.com/2475912/60200945-9737c980-981d-11e9-9cdd-819e89e3a7d6.png">

If you think on a better data structure please let me know.

Thanks!

PD: This is a huge bug I realized just know and that Estrenos V1 has 😱